opsdroid/memory.py

```diff
@@ -20,6 +20,7 @@ class Memory:
     def __init__(self):
         """Create object with minimum properties."""
         self.databases = []
+        self.query_data = {}
```

Comment:
Also needed to add this in the general memory object. I think it could prove handy for other database implementations as well in similar fashion.